### PR TITLE
no colored app list grid borders for now

### PR DESCRIPTION
**Pull request by spasovski**

### CODE REVIEW

**Comment by chuckharmston** [comment]
Without the borders, the spacing looks odd.

    10px      app data      10px   border   10px      app data      10px

looks good, but

    10px      app data      20px      app data      10px

looks imbalanced. I think we probably want to even out each gutter. @pwalm?

File: src/media/css/app-list.styl [hunk 1]
```diff
@@ -129,6 +129,10 @@ $desktop-tile-padding = 20px 50px;
             padding-top: 30px;
         }
     }
+    .app-list-colored .app-list-app,
+    .app-list-colored .app-list-app:nth-child(2n) {
```
Comment (ktngo09) [inline src/media/css/app-list.styl]: Comment the bug for reference

**Comment by ktngo09** [comment]
R+ for the borders. Gutter talk can be later

**Comment by spasovski** [comment]
Don't see a bug number for this.

**Comment by spasovski** [comment]
A better solution was implemented.

**Pull request closed without merging**